### PR TITLE
Resize images/gifs in kedro-viz_visualisation.md

### PR DESCRIPTION
**Pull request by Gundalai-Batkhuu**
## Description
In response to #2643 

## Development notes
- Register the plugin in conf.py
- Add the plugin config dict in conf.py
- Converted inline references of all images in kedro-viz_visualisation.md
- Set the global config for image width to 70%.
- Visually tested the changes by running 'make build-docs'. The results are shown in the original issue #2643.

## Developer Certificate of Origin
We need all contributions to comply with the [Developer Certificate of Origin (DCO)](https://developercertificate.org/). All commits must be signed off by including a `Signed-off-by` line in the commit message. [See our wiki for guidance](https://github.com/kedro-org/kedro/wiki/Guidelines-for-contributing-developers/).

If your PR is blocked due to unsigned commits, then you must follow the instructions under "Rebase the branch" on the GitHub Checks page for your PR. This will retroactively add the sign-off to all unsigned commits and allow the DCO check to pass.

## Checklist

- [x] Read the [contributing](https://github.com/kedro-org/kedro/blob/main/CONTRIBUTING.md) guidelines
- [x] Signed off each commit with a [Developer Certificate of Origin (DCO)](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/managing-the-commit-signoff-policy-for-your-repository)
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added a description of this change in the [`RELEASE.md`](https://github.com/kedro-org/kedro/blob/main/RELEASE.md) file
- [ ] Added tests to cover my changes
- [x] Checked if this change will affect Kedro-Viz, and if so, communicated that with the Viz team

### CODE REVIEW

**Comment by astrojuanlu** [comment]
RTD build failing, you'll need to add https://pypi.org/project/sphinxcontrib-images/ to the list of docs dependencies in `pyproject.toml`

**Comment by stichbury** [comment]
Docs dependencies are in `setup.py` so i've added it there as a commit to the branch so I can see the RTD build to review this.

File: docs/source/conf.py [hunk 1]
```diff
@@ -309,6 +310,14 @@
     )
 ]
 
+# -- Options for Image output ------------------------------------------------
+
+# https://sphinxcontrib-images.readthedocs.io/en/latest/
+images_config = {
+    'override_image_directive': True,
```
Comment (astrojuanlu) [inline docs/source/conf.py:317]: I think this is causing

```
/home/docs/checkouts/readthedocs.org/user_builds/kedro/checkouts/3238/docs/source/index.rst:14:Error in "image" directive: unknown option: "target".
```

File: docs/source/visualisation/kedro-viz_visualisation.md [hunk 2]
```diff
@@ -37,7 +37,9 @@ The command opens a browser tab to serve the visualisation at `http://127.0.0.1:
 
 You should see the following:
 
-![](../meta/images/pipeline_visualisation.png)
+```{eval-rst}
+.. thumbnail:: ../meta/images/pipeline_visualisation.png
+```
```
Comment (astrojuanlu) [inline docs/source/visualisation/kedro-viz_visualisation.md:42]: Maybe let's rework this into MyST

````suggestion
```{thumbnail} ../meta/images/pipeline_visualisation.png
```
````

(haven't tried this syntax yet, please check https://myst-parser.readthedocs.io/en/latest/syntax/roles-and-directives.html)

Comment (Gundalai-Batkhuu) [reply docs/source/visualisation/kedro-viz_visualisation.md]: I've changed the syntax and removed the image_config setting

**Comment by astrojuanlu** [comment]
Rendered result: https://kedro--3238.org.readthedocs.build/en/3238/visualisation/kedro-viz_visualisation.html 

I see a weird behavior on mobile: when I tap the thumbnail the see the full sized image, it gets rendered at the top of the page:

ezgif com-optimize

**Comment by stichbury** [comment]
I think we should close this shortly @Gundalai-Batkhuu unless you plan to do some more to it? We really appreciate the effort but at present we have other priorities in docs and cannot help you diagnose it, so perhaps something to return to later? Please let me know this week or we will close at the end of our sprint this Friday.

**Comment by Gundalai-Batkhuu** [comment]
Hi @stichbury. I haven't found solution to this problem of mobile rendering. Please feel free to close it

**Comment by astrojuanlu** [comment]
Thanks @Gundalai-Batkhuu , closing.

**Pull request closed without merging**